.github/workflows/release-debug-base.yaml

```
@@ -0,0 +1,43 @@
+name: Create Debug Base Image Release
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ 'main', 'feature/**']
```

```suggestion
    branches: ['main']
```

We don't really use feature branches, and if we did we might not want the debug base image rebuilt on every push to them, in case the feature changed the debug base image config in some way.

---

yeah, I forgot to remove it, I just set it for testing purposes for my fork, removed, thank you.